docs/maintain-sync.md

```diff
@@ -6,7 +6,21 @@ sidebar_label: Set up a Full Node
 
 If you're building dapps or products on a Substrate-based chain like Polkadot, Kusama or a custom Substrate implementation, you probably want the ability to run a node-as-a-back-end. After all, it's always better to rely on your own infrastructure than on a third-party-hosted one in this brave new decentralized world.
 
-This guide will show you how to connect to [Kusama network](https://kusama.network), but the same process applies to any other [Substrate](https://substrate.dev)-based chain.
+This guide will show you how to connect to [Kusama network](https://kusama.network), but the same process applies to any other [Substrate](https://substrate.dev)-based chain. First, let's clarify the term _full node_.
+
+### Types of Nodes
+
+A blockchain's growth comes from a _genesis block_, _extrinsics_, and _events_.
+
+When a validator seals block 1, it takes the blockchain's state at block 0. It then applies all pending changes on top of it, and emits the events that are the result of these changes. Later, the state of the chain at block 1 is used in the same way to build the state of the chain at block 2, and so on. Once two thirds of the validators agree on a specific block being valid, it is finalized.
+
+An **archive node** keeps all the past blocks. An archive node makes it convenient to query the past state of the chain at any point in time. Finding out what an account's balance at a certain block was, or which extrinsics resulted in a certain state change are fast operations when using an archive node. However, an archive node takes up a lot of disk space - at Kusama's 1.6 millionth block this was ~20GB. When running a validator, this requirement doubles as [the sentry node](maintain-guides-how-to-setup-sentry-node) in front of a validator should be an archive node too.
```

Comment:
```suggestion
An **archive node** keeps all the past blocks. An archive node makes it convenient to query the past state of the chain at any point in time. Finding out what an account's balance at a certain block was, or which extrinsics resulted in a certain state change are fast operations when using an archive node. However, an archive node takes up a lot of disk space - at Kusama's 1.6 millionth block this was ~13GB. When running a validator, this requirement doubles as [the sentry node](maintain-guides-how-to-setup-sentry-node) in front of a validator should be an archive node too.
```

Estimation was too high compared to the archive node I have running. The difference is significant enough to fix (20 vs. 13 Gb).